### DISCUSSION TITLE
Need help in deploying automations and campaigns from one sfmc instance to other using bldr

### DISCUSSION THREAD

**Discussion by pothangelli123** [basetime/bldr-sfmc#41] [Q&A] [answered]
Hi sir, I'm a sfmc dev working on bldr to deploy automations and campaigns which i have cloned them from my sfmc instance to other sfmc instance. So there is a video in youtube that i followed for most of the usecases using bldr. now i have problem in deploying automations and campaigns.speaking in deplthly my task is to find ways for how to automate a campaign and push to different enviroments. 
@anthonymzupancic

**Comment by anthonymzupancic** [comment, marked as answer]
Hey @pothangelli123,

Thanks for checking out BLDR!

Unfortunately the ability to deploy Automations via the package functionality isn't something we've implemented in BLDR. It was a roadmap item at one point but not sure it's something that will be included in BLDR as we have been working on our AppExchange offering; stashr.io.

Without this turning into a pitch, stashr will very likely have the ability to deploy into separate business units/instances within this year. 

Sorry for any inconvenience, bldr is open source; so if you'd like to take a shot at working in automation studio deployment, you're absolutely welcome to!

**Comment by pothangelli123** [comment]
Sir, i'm very gald that you have responded to me. Its not an inconvenience sir , you have developed a tool which would be helping a lot of sfmc developers and users. And i'm interested to work with bldr sir . I will learn all the prerequisites for contributing to bldr community, as i'm a fresher in working various aspects of sfmc development.Thanks for inviting me
@anthonymzupancic

**Comment by pothangelli123** [comment]
Hi sir, im just commonly using bldr for some task, im getting this error AND sorry for disturbing you for small thing , but the error im getting is
rhg@RHGs-Mac-mini merge % bldr search --as -f Pothan                  
TypeError: Cannot read properties of undefined (reading 'length')
    at /opt/homebrew/lib/node_modules/@basetime/bldr-sfmc/dist/lib/_controllers/_context/automationStudio/index.js:80:92
    at Generator.next (<anonymous>)
    at fulfilled (/opt/homebrew/lib/node_modules/@basetime/bldr-sfmc/dist/lib/_controllers/_context/automationStudio/index.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
 Can you please help me in solving this error
@anthonymzupancic
